### PR TITLE
Todd fix promote channel

### PR DESCRIPTION
**Pull request by tdensmore**
Small change so that the promote command:

`replicated release promote 14 todd --version 0.0.14`

will output human readable channel name:

`Channel todd successfully set to release 14`

instead of the current:

`Channel 1lTUjrHZchcezP6lUWhBwrdMcbg successfully set to release 14`

### CODE REVIEW

**Comment by tdensmore** [comment]
Fixed the bad error message.

**Comment by tdensmore** [comment]
Let me know if this looks good to you @laverya

File: pkg/kotsclient/app.go [hunk 1]
```diff
@@ -105,5 +105,5 @@ func (c *GraphQLClient) GetApp(appID string) (*types.App, error) {
 		}
 	}
 
-	return nil, errors.New("App not found")
```
Comment (laverya) [inline pkg/kotsclient/app.go:108]: I don't like this (string concat in general)

I'd prefer for this to be `fmt.Errorf("app %q not found", appID)` or similar instead, but it's not a blocker

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ logs/
 .DS_Store
 bin/
 vendor/
+.idea
```
Comment (laverya) [inline .gitignore:8]: I just add this to my global gitignore - see https://stackoverflow.com/a/7335487/5657035

**Review by laverya — APPROVED** [review]

**Pull request merged**